### PR TITLE
INT-6772 Adding Operator Intallation Suggestion

### PR DESCRIPTION
**Pull request by hectorhuol**
Updating docs to add Operator Installation Suggestion to handle unscheduled downtime due to new operator releases.

JIRA: https://issues.sonatype.org/browse/INT-6772

### CODE REVIEW

File: scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml [hunk 1]
```diff
@@ -175,6 +175,24 @@ spec:
 
     This operator will be released on a quarterly basis.
 
+    ## Operator Intallation Considerations
```
Comment (jflinchbaugh) [inline scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml:178]: typo: "Installation"

File: scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml [hunk 2]
```diff
@@ -175,6 +175,24 @@ spec:
...
+    If you want to avoid this unscheduled downtime, please consider the next setup 
+    for the operator installation:
+    
+    * **Update Channel:** _stable_
+    * **Installation mode:** _A specific namespace on the cluster_
+    * **Installed Namespace:** _Create a new one to install the operator_
+        * You may also select an existing namespace, but as we are suggesting moving to a
+        manual approval, this will affect all the operators on the selected namespace
+    * **Update approval:** _Manual_
+        * Remeber that moving the approval to **Manual** will affect all the operators in
```
Comment (jflinchbaugh) [inline scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml:193]: typo: "Remember"

File: scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml [hunk 3]
```diff
@@ -175,6 +175,24 @@ spec:
...
     This operator will be released on a quarterly basis.
 
+    ## Operator Intallation Considerations
+    
+    If you use the default configuration for the Nexus Repository Operator installation, 
+    please notice that on any new operator release, the corresponding deployments are 
+    also updated without user intervention, resulting in unscheduled downtime.
+
+    If you want to avoid this unscheduled downtime, please consider the next setup 
+    for the operator installation:
```
Comment (jflinchbaugh) [inline scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml:185]: I wonder if the user could merely be told: "we recommend installing the operator into its own namespace with manual approval for updates", and they'd figure out these settings below. I often figure the less I say, the less that falls out of date for UI changes.

I might consider shorter and more direct sentences in general to make it more scannable, but that might just be my style.

File: scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml (hunk 1 above)
Comment (jflinchbaugh) [reply scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml:178]: I like this placement in the tree of titles. 👍

File: scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml (hunk 3 above)
Comment (hectorhuol) [reply scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml:185]: You are right, being this specific may require more time for maintenance, as the UI may change, I am going to update the message with the sentence you shared, I like it more.

File: scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml (hunk 1 above)
Comment (hectorhuol) [reply scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml]: Done

File: scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml (hunk 2 above)
Comment (hectorhuol) [reply scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml]: Oops, done 😅

File: scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml (hunk 3 above)
Comment (hectorhuol) [reply scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml]: Done. I also applied the changes for the `operator-iq` repo

File: scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml [hunk 4]
```diff
@@ -175,6 +175,15 @@ spec:
...
     This operator will be released on a quarterly basis.
 
+    ## Controlling Automatic vs Manual Update
+    
+    If you use the default configuration for the Nexus Repository Operator installation, 
+    please notice that on any new operator release, the corresponding deployments are 
+    also updated without user intervention, resulting in unscheduled downtime.
+
+    If you want to avoid this unscheduled downtime, we recommend installing the operator 
+    into **its own namespace** with **manual approval** for updates.
```
Comment (jflinchbaugh) [inline scripts/templates/nxrm-operator-certified.clusterserviceversion.yaml:185]: I'd totally read this! 🎉

**Review by jflinchbaugh — APPROVED** [review]
+1

**Review by eduard-tita — APPROVED** [review]

**Pull request merged**